samples/gcp-storage/src/hello.js

```diff
@@ -0,0 +1,33 @@
+const { App, HTTPBindingMiddleware, StorageMiddleware } = require('core');
+const { GcpModule } = require('gcp');
+
+const app = new App(new GcpModule());
+
+async function streamToString(readableStream) {
```

Comment:
Perhaps you could move this function to another file? To focus on the write and read functions of the GCP Implementation